tests/test_examples.py

```diff
@@ -64,6 +64,7 @@ def get_solve_with_gurobipy(self) -> Optional[Any]:
         skip_solvers=["highs"],  # Has quadratics
     ),
     Example("sudoku"),
+    Example("pumped_storage"),
```

Comment:
does this example have a unique solution? If not, it needs to be marked as `unique_solution=False`